### PR TITLE
Assets

### PR DESCRIPTION
**Pull request by cboesch-gpsw**

### CODE REVIEW

File: pynodegl-utils/pynodegl_utils/assets/CREDITS.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## `mire-h264.mp4`
+
+Generated with `ffmpeg -f lavfi -i testsrc=d=15:r=60:s=hd720 -preset placebo -c:v h264 mire-h264.mp4`
+
+## `mire-hevc.mp4`
+
+Generated with `ffmpeg -f lavfi -i testsrc2=d=30:r=30 -preset placebo -c:v hevc mire-hevc.mp4`
+
+## `Unsplash_Michael-Anfang_Rooster_cropped.jpg`
```
Comment (mbouron-gpsw) [inline pynodegl-utils/pynodegl_utils/assets/CREDITS.md:11]: Why using `-` in that particular place ? Same question for fallen-leaf ? Can we stick with only `_` ?

**Comment by mbouron-gpsw** [comment]
Other than my comment, LGTM.

**Review by mbouron-gpsw — APPROVED** [review]

File: pynodegl-utils/pynodegl_utils/assets/CREDITS.md (hunk 1 above)
Comment (cboesch-gpsw) [reply pynodegl-utils/pynodegl_utils/assets/CREDITS.md]: Here are the new names:

```
OpenMoji-1F342-Fallen_Leaf.png
OpenMoji-1F439-Hamster.png
Unsplash-Michael_Anfang-Rooster-cropped.jpg
Unsplash-Romane_Gautun-Red_Panda-cropped.jpg
```

Logic is "Source-ID-Title" and "Source-Author-Title-ExtraNote", then within each group we use `_` to differentiate from the group split character (`-`).

Comment (mbouron-gpsw) [reply pynodegl-utils/pynodegl_utils/assets/CREDITS.md]: Ok.

**Pull request merged**